### ISSUE TITLE
voice message and music have same speed → should not have same speed

### ISSUE THREAD

**Issue by alerezaaa** [overtake/TelegramSwift#954]
Hi there

### Issue:

I listen to a voice message and I speed up voice playing speed to 1.5 or sth, After a while, I start to listen a music but the music start playing with speed 1.5x (or whatever I set for voice message)

### Expected:

As Telegram-desktop, it has been expected to paly music and voice message completely independant ⇒ voice at speed 1.5x (or whatever) and music starts at speed 1x

In other words, Music and voice-message settings should be independant

thanks in advance, I hope, I could help as its open source

**Comment by overtake** [comment]
you right. the fix will be available today.
